boards/FSM/FSM.ioc

```diff
@@ -225,10 +226,9 @@ PA6.GPIO_Label=BRAKE_PRESS_BUFF
 PA6.Locked=true
 PA6.Mode=IN3-Single-Ended
 PA6.Signal=ADC2_IN3
-PA7.GPIOParameters=PinState,GPIO_Label
+PA7.GPIOParameters=GPIO_Label
 PA7.GPIO_Label=STATUS_G
 PA7.Locked=true
-PA7.PinState=GPIO_PIN_SET
```

Comment:
Seems to be a change with the pin state here, can you make sure that we change this back to what it was before

Reply:
not sure if these are introduced by u or by cube lol